templates/data/opensearch/index.html

```diff
@@ -0,0 +1,506 @@
+                        alt="Kubernetes",
+                        width="105",
+                        height="384",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/165fa040-openstack-logo.png",
+                        alt="OpenStack",
+                        width="426",
+                        height="256",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/a89f2843-microcloud.png",
+                        alt="Microcloud",
+                        width="394",
+                        height="312",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Easy to deploy and manage OpenSearch</h2>
+      </div>
+      <div class="col">
+        <div class="p-image-wrapper">
+          <div class="p-image-container u-hide--small">
+            {{ image(url="https://assets.ubuntu.com/v1/348fac82-Image.png",
+                        alt="",
+                        width="1800",
+                        height="1015",
+                        hi_def=True,
+                        attrs={"class": "p-image-container__image"},
+                        loading="lazy") | safe
+            }}
+          </div>
+        </div>
+        <p>
+          Operating and supporting a production-grade OpenSearch solution can be complex and challenging. Canonical's Charmed OpenSearch automates operational tasks so you can spend more time on value-driving activities.
+        </p>
+        <hr class="is-muted" />
+        <div class="p-section--shallow">
+          <ul class="p-list--divided">
+            <li class="p-list__item is-ticked">Automated security and configuration best practices built in</li>
+            <li class="p-list__item is-ticked">Made for physical and virtual machines (VMs)</li>
+            <li class="p-list__item is-ticked">Integrated observability for monitoring, alerting and auditing</li>
+            <li class="p-list__item is-ticked">Multi-cloud support</li>
+            <li class="p-list__item is-ticked">Self-healing</li>
+          </ul>
+          <hr class="is-muted" />
+          <p>
+            <a href="/data/opensearch/managed">Explore Managed OpenSearch Solutions&nbsp;&rsaquo;</a>
```

Comment:
This link 404s

Reply:
This page is still in development hence this branch merges to feature and not to main